tests/suites/0_stateless/02_0039_function_strings_find_in_set.sql

```diff
@@ -0,0 +1,25 @@
+SELECT '=== const ===';
+SELECT FIND_IN_SET('a', 'a');
```

Comment:
It‘s a little hard to track the results and test cases in order.

I suggest using 

```
SELECT FIND_IN_SET('a', 'a') = 1;
SELECT FIND_IN_SET('a', ',a') = 2;
SELECT FIND_IN_SET('a', '1,2,a,4,5') = 1;
```